Marketplace/controllers/api/apiProductController.js

```diff
@@ -1,11 +1,77 @@
-const db = require("../../database/models");
-const entidad = db.Producto;
+const productoRepository = require("../../repositories/productoRepository");
+const categoriaRepository = require("../../repositories/categoriaRepository");
 
 module.exports = {
-	listado: (req, res) => {
-
+	listado: async (req, res) => {
+		let listado = null;
+		listado = await productoRepository.ObtenerTodas();
```

Comment:
Esto podria ser una sola linea. Yo le pondria otro nombre a la variable, para que sea mas claro que es
```suggestion
		let listadoProductos = await productoRepository.ObtenerTodas();
```

Reply:
Christian,
- A partir de tu comentario, recién le cambié el nombre de _listado_ a _data_. Lo vas a ver cuando actualice el pull.
- Mi intención es estandarizar. Por eso le puse el mismo nombre (data) las 3 veces. Para que quedara en claro dónde se toma la información, más allá de que todos sabemos. Me parece que así, ayuda a la lectura del código. A mí, por lo menos me ayuda. Así como vos usaste entidad en todos los repositorios, creo que es una manera simple de estandarizar luego el código entre las diferentes tablas.
- Por esa razón también usé el let. Si bien como vos decís podría ser una sola línea, y yo soy "pro" eso, lo dejé así para que quedara estandarizada la línea del await. Si se leyera una sola vez en cada método del controller, entonces sí lo haría en una sola línea.
- Respecto a que se entienda qué se está leyendo, lo dejo en mano del nombre del repositorio, que está en la misma línea.